.golangci.yml

```diff
@@ -17,7 +17,8 @@ linters:
     - gofmt
     - goimports
     - golint
-    - gosimple
+    # Fixme: Broken? dies w/o error randomly, between ~100-200 seconds
+    # - gosimple
```

Comment:
beauty, love random fails.

Reply:
:cry: this is not perfect either, it still randomly takes like 10-minutes to run all these checks.  But after all my iterations here since commenting out `gosimple`, there have been no more random no-error failures.

Reply:
I improved this comment.